src/test/resources/test-cases/RMLTC__NS__-XML/student-partially-NS.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+
+<!-- This is a `partially namespaced tree`. Name,Address,... use default NS -->
```

Comment:
You mentioned that the partially namespaced tree wasn't investigated yet.
The files are here, could you try to add the RML mapping rules so we have a full test case as well?
Maybe it is not hard to support partially namespaced tree as well?